### DISCUSSION TITLE
Use `panvimdoc`

### DISCUSSION THREAD

**Discussion by lopi-py** [nvim-neo-tree/neo-tree.nvim#244] [Ideas]
https://github.com/kdheepak/panvimdoc
> Writing documentation is hard. Writing documentation in vimdoc for vim plugins is an additional hassle. Making writing vim plugin documentation frictionless is important.
> 
> Writing documentation in markdown and converting it to vimdoc can help toward that goal. This way, plugin authors will have to write documentation just once ( for example, as part of the README of the project ), and the vim documentation can be autogenerated.

**Comment by cseickel** [comment]
I went around and around with this when I started the project. The end result was that I decided to just make the vim help file be the authoritative reference for **actual** users, and use the README as a highlight/sales pitch of features and an intro to usage for _potential_ users.

I felt like whichever one was auto generated would be lacking, and the hassle of working with the transpiler and learning the syntax was unattractive to me.

Of course, documentation could be better. If you or someone else is willing to revamp that documentation and configure everything then I will happily maintain it in that format.
